hw/ip/sram_ctrl/dv/cov/sram_ctrl_cov_excl.el

```diff
@@ -0,0 +1,50 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: udij
+// Format Version: 2
+// Date: Wed May  5 10:09:25 2021
+// ExclMode: default
+//==================================================
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[0].u_prim_alert_sender_parity.i_decode_ping
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[0].u_prim_alert_sender_parity.i_decode_ack
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[1].u_prim_alert_sender_parity.i_decode_ping
+CHECKSUM: "3767637819"
+ANNOTATION: "[EXTERNAL] alert_diff_decoder is fully verified by FPV testbench"
+INSTANCE:tb.dut.u_sram_ctrl.gen_alerts[1].u_prim_alert_sender_parity.i_decode_ack
+CHECKSUM: "2966200294"
+ANNOTATION: "[EXTERNAL] prim_lfsr is verified in separate testbench"
+INSTANCE:tb.dut.u_ram1p_sram.u_lfsr
+CHECKSUM: "3927569706"
+ANNOTATION: "[EXTERNAL] prim_prince is verified in separate testbench"
+INSTANCE:tb.dut.u_ram1p_sram.gen_par_scr[0].u_prim_prince
+CHECKSUM: "1684945168 1445297464"
```

Comment:
These also belong to the cover cfg. The IO ports of coverage-black-boxed entities must not be excluded.

Reply:
just to clarify, you mean the `common_cov_excl.cfg` ?